src/libraries/Microsoft.PowerFx.Core/Public/Types/InvalidType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx.Core.Public.Types
+{
+    public class InvalidType : FormulaType
+    {
+        internal InvalidType()
+            : base(DType.Invalid)
+        {
+        }
+
+        public override void Visit(ITypeVistor vistor)
+        {
+            throw new NotImplementedException();
```

Comment:
Should be implemented in ITypeVisitor.cs

Reply:
Done